packages/darklang/stdlib/noModule.dark

```diff
@@ -0,0 +1,12 @@
+module Darklang =
+  module Stdlib =
+    /// Returns true if the two value are equal
+    let equals (a: 'a) (b: 'a) : Bool = Builtin.equals a b
+
+
+    /// Returns true if the two value are not equal
+    let notEquals (a: 'a) (b: 'a) : Bool = Builtin.notEquals a b
+
+
+    /// Unwrap an Option or Result, returning the value or a DError if None
+    let unwrap (value: 'a) : 'a = Builtin.unwrap value
```

Comment:
TODO: check if this can be rewritten in dark

Reply:
I'm wondering if we should even expose this, or if it should only exist as a `!` once we have darklang syntax for it?

Reply:
It can't be rewritten in Dark though